src/components/button/button.stories.tsx

```diff
@@ -42,6 +42,13 @@ export const StandardAttributes = {
   },
 }
 
+/**
+ *
+ * **This option will be removed from `Button` in a future version.**
+ * And it will sever as a new component `ButtonGroup`.
```

Comment:
**quibble:** I find the first sentence here a little confusing. Doesn't seem clear that we're talking about the `ButtonGroup` component exported from the `/button` folder vs the `ButtonGroup` component exported from `/button-group` 🤔 

**suggestion:** Maybe we can simply remove this story from Button's docs altogether?

Reply:
I agree, I will update the PR by removing `ButtonGroup` altogether from `/button`